src/components/WwwFrame/EndOfYearAppealBanner/AppealBanner.vue

```diff
@@ -101,19 +93,68 @@ export default {
 	data() {
 		return {
 			open: true,
+			appealEnabled: false,
+			appealMatchEnabled: false,
+			appealBonusEnabled: false,
+			isAppealShrunk: false,
 		};
 	},
 	apollo: {
 		query: appealBannerQuery,
 		preFetch: true,
 		result({ data }) {
-			this.isVisible = _get(data, 'general.setting.value') === 'true' || false;
+			this.appealEnabled = settingEnabled(
+				data,
+				'general.appeal_enabled.value',
+				'general.appeal_start_time.value',
+				'general.appeal_end_time.value'
+			);
```

Comment:
Check enabled + start and end times.

Reply:
This makes more sense than what was in there. 🧠Looking at all the setting options.